mobile/src/main/res/values/strings.xml

```diff
@@ -4,7 +4,7 @@
     <!-- See: https://github.com/openhab/openhab-android/blob/master/README.md#localization -->
 
     <!-- General app strings -->
-    <string name="openhab_service_type" translatable="false">_openhab-server._tcp.local.</string>
+    <string name="openhab_service_type" translatable="false">_openhab-server-ssl._tcp.local.</string>
```

Comment:
This doesn't seem to be complete. You need to change `http` to `https` where the url is built, similar to https://github.com/openhab/openhab-android/pull/496/files#diff-6f29e9c52ba079a271b8efc27cfe5468L156

Reply:
Dann, right. I'll fix it.